boards/microduino-corerf/Makefile.include

```diff
@@ -0,0 +1,16 @@
+# configure the terminal program
+PORT_LINUX  ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+BAUD        ?= 57600
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# PROGRAMMER defaults to UM232H which is a FT232H breakout board
+# externally connected using wires
+PROGRAMMER ?= UM232H
```

Comment:
If you want to me super happy, you can declare this like that:

``` Makefile
PROGRAMMER_MICRODUINO_CORERF ?= UM232H
PROGRAMMER ?= $(PROGRAMMER_MICRODUINO_CORERF)
```

But that is an optional suggestion. (Squash right away, if you decide to do this.)

Reply:
Of course I want you to be happy - but what is the purpose of this?

Reply:
I could add `export PROGRAMMER_MICRODUINO_CORERF=dragon_isp` to my `.profile`. If I would add `PROGRAMMER=dragon_isp`, I would have trouble flashing ARM boards with a configurable programmer.